streams/src/test/java/org/apache/kafka/streams/integration/StandbyTaskEOSIntegrationTest.java

```diff
@@ -112,6 +118,19 @@ public void createTopics() throws Exception {
         CLUSTER.createTopic(outputTopic, 1, 3);
     }
 
+    @After
+    public void cleanUp() {
+        if (streamInstanceOne != null) {
+            streamInstanceOne.close();
+        }
+        if (streamInstanceTwo != null) {
+            streamInstanceTwo.close();
+        }
+        if (streamInstanceOneRecovery != null) {
+            streamInstanceOneRecovery.close();
+        }
```

Comment:
There are no logical changes to this test, I just had to refactor it a bit because we were creating two copies of the same KafkaStreams at the same time (with the same app.dir & state.dir), even though one of them wasn't started until much later. Since we do the state initialization inside the KafkaStreams constructor, this was no good